controllers/common/node_info.py

```diff
@@ -46,6 +46,13 @@ def _get_iter(self):
     def __iter__(self):
         return self._get_iter()
 
+    def get_by_connectivity_type(self, connectivity_type):
+        return {
+            array_config.NVME_OVER_FC_CONNECTIVITY_TYPE: self.nvme_nqns,
+            array_config.FC_CONNECTIVITY_TYPE: self.fc_wwns,
+            array_config.ISCSI_CONNECTIVITY_TYPE: self.iscsi_iqns
+        }[connectivity_type]
```

Comment:
```suggestion
        return {
            array_config.NVME_OVER_FC_CONNECTIVITY_TYPE: self.nvme_nqns,
            array_config.FC_CONNECTIVITY_TYPE: self.fc_wwns,
            array_config.ISCSI_CONNECTIVITY_TYPE: self.iscsi_iqns
        }.get(connectivity_type)
```
please move this dict creation to `__init__` so that `_get_iter` could use it (with `.items()`)

Reply:
done - #561